client/blocks/importer/wordpress/content-chooser/index.tsx

```diff
@@ -1,153 +0,0 @@
-import { NextButton, SelectItems } from '@automattic/onboarding';
-import { useI18n } from '@wordpress/react-i18n';
-import clsx from 'clsx';
-import React, { useEffect, useRef, useState } from 'react';
-import illustrationImg from 'calypso/assets/images/onboarding/import-2.svg';
-import ActionCard from 'calypso/components/action-card';
-import FormattedHeader from 'calypso/components/formatted-header';
-import { preventWidows } from 'calypso/lib/formatting';
-import wpcom from 'calypso/lib/wp';
-import { jetpack } from 'calypso/signup/icons';
-import { useDispatch } from 'calypso/state';
-import { recordTracksEvent } from 'calypso/state/analytics/actions';
-import { requestSite } from 'calypso/state/sites/actions';
-import type { SiteDetails } from '@automattic/data-stores';
-
-import './style.scss';
-/* eslint-disable wpcalypso/jsx-classname-namespace */
-
-const trackEventName = 'calypso_signup_actions_submit_step';
-const trackEventParams = {
-	intent: 'import',
-	step: 'importReadyPreview',
-};
-
-interface Props {
-	siteId: number;
-	siteSlug: string;
-	fromSite: string;
-	onJetpackSelection: () => void;
-	onContentOnlySelection: () => void;
-	onContentEverythingSelection: () => void;
-}
-
-export const ContentChooser: React.FunctionComponent< Props > = ( props ) => {
-	const { __ } = useI18n();
-	const dispatch = useDispatch();
-	const { fromSite, onJetpackSelection, onContentOnlySelection, onContentEverythingSelection } =
-		props;
-
-	/**
-	 ↓ Fields
-	 */
-	const showJetpackConnectionBlock = !! fromSite;
-	const [ hasOriginSiteJetpackConnected, setHasOriginSiteJetpackConnected ] = useState( false );
-	const [ initialFetching, setInitialFetching ] = useState( true );
-	const firstRun = useRef( true );
-
-	/**
-	 ↓ Effects
-	 */
-	useEffect( () => {
-		checkOriginSiteJetpackConnection();
-		const interval = setInterval( checkOriginSiteJetpackConnection, 5000 );
-
-		return () => clearInterval( interval );
-	}, [] );
-
-	useEffect( () => {
-		if ( firstRun.current ) {
-			firstRun.current = false;
-			return;
-		}
-
-		dispatch( requestSite( fromSite ) );
-	}, [ hasOriginSiteJetpackConnected ] );
-
-	useEffect( () => {
-		dispatch( recordTracksEvent( trackEventName, trackEventParams ) );
-	}, [] );
-
-	/**
-	 ↓ Methods
-	 */
-	function checkOriginSiteJetpackConnection() {
-		if ( ! fromSite ) {
-			return;
-		}
-
-		wpcom
-			.site( fromSite )
-			.get( { apiVersion: '1.2' } )
-			.then( ( site: SiteDetails ) =>
-				setHasOriginSiteJetpackConnected( !! ( site && site.capabilities ) )
-			)
-			.catch( () => setHasOriginSiteJetpackConnected( false ) )
-			.finally( () => setInitialFetching( false ) );
-	}
-
-	return (
-		<div className={ clsx( 'import-layout', 'content-chooser' ) }>
```

Comment:
Seems like we have opportunities for some e2e cleanup here:

https://github.com/Automattic/wp-calypso/blob/c6d0aad853323f841c490f2ecbdb8e4f2c5069dd/packages/calypso-e2e/src/lib/flows/start-import-flow.ts#L15

Reply:
True, although I preferred not to touch the e2e test code until I have more clarity on the expected behaviour: https://github.com/Automattic/wp-calypso/pull/102282#discussion_r2022742688